node/src/chain_spec.rs

```diff
@@ -173,7 +173,62 @@ pub fn remote_testnet_config() -> Result<ChainSpec, String> {
 		Some("Golden Gate"),
 		None,
 		// Properties
-		properties(),
+		properties("GGXT"),
+		// Extensions
+		None,
+	))
+}
+
+#[cfg(not(feature = "testnet"))]
+pub fn sydney_testnet_config() -> Result<ChainSpec, String> {
+	use sc_telemetry::TelemetryEndpoints;
+
+	let wasm_binary = WASM_BINARY.ok_or_else(|| "Runtime wasm not available".to_string())?;
+
+	Ok(ChainSpec::from_genesis(
+		"GGX Sydney Testnet",
+		"GGX",
+		ChainType::Live,
+		move || {
+			testnet_genesis(
+				wasm_binary,
+				// Sudo account
+				sr25519::Public::from_ss58check("5EHkPQgHPKLT4XTEkZcVWpwvLziBS3Qf2oUg94YAk79YVFdw")
+					.unwrap()
+					.into_account()
+					.into(),
+				// Pre-funded accounts
+				vec![sr25519::Public::from_ss58check(
+					"5EHkPQgHPKLT4XTEkZcVWpwvLziBS3Qf2oUg94YAk79YVFdw",
+				)
+				.unwrap()
+				.into_account()
+				.into()],
+				// Initial Validator
+				vec![ValidatorIdentity::from_pub(
+					"5GWHWMD1eFZkkZZ2XRMSwhsbdXhwirfKHJm4LYh66khuwxgT",
+					"5EHkPQgHPKLT4XTEkZcVWpwvLziBS3Qf2oUg94YAk79YVFdw",
+					"KW5iGDUzxmzxkRHjXvMALXQqLfX1WCi7wVbWTCHHZfykuNKYt", //subkey generate --scheme ecdsa, Public key (SS58)
+				)],
+				888888,
```

Comment:
What is this? @akorchyn

Reply:
Copy paste, as a mock up.

Reply:
@akorchyn I think we can come up with a nice ID, if 888888 is an id. Not sure if there are any specific rules for that.

Reply:
It's evm chian id. Need @xcthulhu correction what id will be for EVM, and what for prefix